go-api/main.go

```diff
@@ -1,6 +1,7 @@
 package main
 
 import (
+	_ "embed"
```

Comment:
I haven't noticed the `embed` before. I am curious, what is it for?

Reply:
following is version string declared in `main.go`
```
//go:embed .version
var version string
```

when executable file (which is built through `go build` command) is ran,
file reference such as .version in here breaks since external files aren't embedded inside executable, and it fails to find file in the path.

you can look for more details here
https://pkg.go.dev/embed